packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -95,6 +95,12 @@ class CreateCommand extends FlutterCommand {
       defaultsTo: null,
       valueHelp: 'id',
     );
+    argParser.addOption(
+      'list-samples',
+      help: 'Specifies a file to output a list of Flutter code samples that can '
+        'be created with --sample to a JSON file.',
```

Comment:
Maybe:

    'Specifies a JSON output file for a listing of Flutter code samples that can created with --sample.'

?

Reply:
Done!